index.html

```diff
@@ -1712,16 +1712,6 @@ <h2>Security Vocabulary Definitions</h2>
       given in this specification share many similarities with OpenAPI, they are not compatible.
       </p>
 
-      <p>
-      If a value of <code>auto</code> is set for the <code>in</code> field of a 
-      <code>SecurityScheme</code>, then the <code>name</code> field SHOULD NOT
-      be set.
-      In this case, the application of the <code>SecurityScheme</code> MUST
-      follow the respective specification for the given protocol (e. g. 
-      [[!RFC8288]] when using the <code>BasicSecurityScheme</code> with 
-      HTTP).
-      </p>
-
```

Comment:
That would be logical, but is a pain because of the way rendering works (the text would have to go in to the ontology files).  Let's get the text right first, then I can move it if it makes sense.